engine/runtime/src/main/java/org/enso/interpreter/runtime/callable/atom/Atom.java

```diff
@@ -203,6 +205,25 @@ boolean hasFunctionalDispatch() {
     return true;
   }
 
+  @ExportMessage
+  boolean isDate(
```

Comment:
This looks inconsistent with google-java-format, which is the requirement for this project.

Reply:
Related to #3394 - once there is a way to run the proper formatter and the gate fails - the proper formatting becomes a requirement. I'll spend some time trying to _guess the right fix_, but if I fail, we can probably wait for #3394 to turn the check  on and fix all the problems in a batch.